artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/ClusterManager.java

```diff
@@ -542,15 +542,15 @@ public void clear() {
          try {
             bridge.stop();
          } catch (Exception e) {
-            ActiveMQServerLogger.LOGGER.warn(e.getMessage(), e);
```

Comment:
Any catch (Exception e) log.. should just use logger.warn.. maybe the inidivualized logger.. but do we really need individualized loggers for everything?

Reply:
It's a good example. How critical it is for us to log bridge.stop failure ? What is the probability that this will happen? Maybe it's better for gss issue to have individual id for this case?

Reply:
Removing logging in case of exception is never good. Else you run blind

Reply:
agree !

Reply:
what I'm getting at is here we catch if bridge.stop errors and log, but in the same code in other places we don't log anything when bridge.stop occurs.

Reply:
yeap, but here we have 3 options howto log it 
* logger.warn(e.getMessage(), e);
* ActiveMQServerLogger.LOGGER.warn(e.getMessage(), e);
* ActiveMQServerLogger.LOGGER.failedToStopBridge(e);
What is the best in this case and why ?

Reply:
On this case it's better logger.warn(e.getMessage(), e);


failedToStopBridge(e) wouldn't buy us much in terms of extra information.


ActiveMQServerLogger.LOGGER would somewhat hide where it's happening.. although you would have the stack trace... but I would prefer logger.warn...

I'm a bit concerned about poluting the Logger with too many log codes.. it's already complicated to manage it.